README.md

```diff
@@ -33,30 +33,39 @@ Here is how static analysis is used internally at Google:
 
 ## Enabled Lints
 
-The currently enabled lints can be found in the sample
-[analysis_options.yaml](https://github.com/dart-lang/pedantic/blob/master/lib/analysis_options.yaml).
+The currently enabled lints can be found in
+[analysis_options.1.7.0.yaml](https://github.com/dart-lang/pedantic/blob/master/lib/analysis_options.1.7.0.yaml).
```

Comment:
Here, do you want to reference a specific version, or just the main `analysis_options.yaml`?

Reply:
I want the specific version so that people can click through and see the lints.